ui/package.json

```diff
@@ -60,6 +59,9 @@
     "whatwg-fetch": "2.0.3",
     "why-did-you-update": "^0.1.1"
   },
+  "optionalDependencies": {
```

Comment:
What is this for?

Reply:
[fsevents](https://www.npmjs.com/package/) is a mac only library for watching file-system events. Useful for hot-reload - not useful for straight build, and it causes errors when trying to install on other distros:

```
npm ERR! code EBADPLATFORM npm ERR! notsup Unsupported platform for
fsevents@1.1.3: wanted {"os":"darwin","arch":"any"} (current:
```

Because building the seqr UI bundles up all the dependencies, maybe `dependencies` could mean anything required to build, and `devDependencies` could mean any specific dev packages. Eg: I need babel to build, but it's currently listed as a devDependency.